src/GlobalStyles.js

```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500;700&display=swap');
+  
+  * {
+    box-sizing: border-box;
+  }
+
+  body {
+    margin: 0;
```

Comment:
You could also reset the padding to `padding: 0;` to be sure you start from a clean slate :)